recipe/meta.yaml

```diff
@@ -58,6 +58,7 @@ outputs:
       files:
         - mpiexec.sh
         - tests/helloworld.sh
+        - mpiexec
```

Comment:
This is wrong. We do not have a file named "mpiexec" in the recipe directory. What are you trying to do?

Reply:
FYI, this is from my home workstation, running MPICH from Fedora 34 (system package, not conda forge)

```
$ mpichversion | head -n 3
MPICH Version:    	3.4.1
MPICH Release date:	Fri Jan 22 14:17:48 CST 2021
MPICH Device:    	ch3:nemesis

$ mpiexec
Segmentation fault (core dumped)
```

Indeed, a bare `mpiexec` invocation without args segfaults. This is most likely a bug in MPICH and it is not related conda-forge build/packaging.

@hzhou FYI.

Reply:
Yeah, I got the same thing. If we can conclude that this is simply a bug in mpich, perhaps we can just close #61, possibly with a link to the relevant mpich ticket.